### PR TITLE
(Logger) Rename package to log

### PR DESCRIPTION
**Pull request by xtda**
# PR Description

Just a quick small change that renames the logger package to "log" to stop the requirement of adding logger infront of the import that has annoyed me for awhile now :D

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Travis with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: engine/engine.go [hunk 1]
```diff
@@ -525,6 +525,6 @@ func (e *Engine) Stop() {
 	e.ServicesWG.Wait()
 	err := gctlog.CloseLogger()
 	if err != nil {
-		log.Printf("Failed to close logger. Error: %v\n", err)
+		log.Printf("Failed to close log. Error: %v\n", err)
```
Comment (thrasher-) [inline engine/engine.go]: logger

File: log/logger_setup.go [hunk 2]
```diff
@@ -63,7 +63,7 @@ func GenDefaultSettings() (log Config) {
 func configureSubLogger(logger, levels string, output io.Writer) error {
 	found, logPtr := validSubLogger(logger)
 	if !found {
-		return fmt.Errorf("logger %v not found", logger)
+		return fmt.Errorf("log %v not found", logger)
```
Comment (thrasher-) [inline log/logger_setup.go]: sublogger (or logger) %v not found

File: engine/engine.go (hunk 1 above)
Comment (xtda) [reply engine/engine.go]: :D vscode doesn't have as smart refactoring as Goland thought i picked up all these changes !

reverted this and the other instance

Comment (thrasher-) [reply engine/engine.go]: No problemo, that's why we review things 😉

**Pull request merged**